canvas_site_creator/templates/canvas_site_creator/base.html

```diff
@@ -7,9 +7,16 @@
 <head>
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
     <link rel="stylesheet" type="text/css" href="{% static 'font-awesome-4.5.0/css/font-awesome.min.css' %}">
-    <link rel="stylesheet" href="{% static 'lti-css/bulkSiteCreator.css' %}">
     <link rel="stylesheet" href="{% static 'canvas_site_creator/css/app.css' %}">
-    {% block css %}{% endblock css %}
+    <link rel="stylesheet" type="text/css" href="{% static 'bootstrap-3.3.6/dist/css/bootstrap.min.css' %}" media="screen"/>
+
+    {% block css %}
+      {% if debug %}
+        <link rel="stylesheet" type="text/css" href="{% static 'bootstrap-3.3.6/dist/css/bootstrap.css' %}" media="screen"/>
+      {% else %}
+        <link rel="stylesheet" type="text/css" href="{% static 'bootstrap-3.3.6/dist/css/bootstrap.min.css' %}" media="screen"/>
+      {% endif %}
+    {% endblock css %}
 {% if debug %}
```

Comment:
As per the issue Chrissy reported, if we get the bootstrap JS into this block, and remove the tooltip.js and tooltip.css stuff from the course selection page, that appears to fix the weird modal behavior.